region/norfair/west/Crumble Shaft.json

```diff
@@ -281,6 +281,10 @@
       "note": [
         "Jump and aim down to lower the camera so the middle platform Sova starts moving.",
         "Then freeze it by aiming down, shoot the shot block, and use the Sova as a stable platform."
+      ],
+      "devNote": [
+        "Wall jumping and using the same frozen Sova, or falling onto the Sova just below the item and then jumping to the top Sova could be added,",
+        "but this would require an obstacle to ensure they haven't fallen or died, and is not really easier than a crumble jump."
       ]
```

Comment:
this felt too complex for hard, and the crumble jump (VH) was easier than this, but I think it's just getting back without falling that would push it up. 
Most players would probably just do the crumble jump here, and you kind of have multiple attempts, so it feels fine to stay in hard.
I was also concerned of if the Sova's were moved or killed, but maybe its not too likely. Freezing the Sova below while falling with two quick shots was also a little precise/unintuitive.

Reply:
- I haven't seen this used much
- Falling is punishing enough  that there are situations where you would take the time to do the more reliable strat.  Many people die checking crumble shaft in vanilla map races.
- The ice platforms seem more important if you don't have walljump or the runway to jump directly to the item.  Walljumping off the crumbles is safer than crumble jumping.